playground/backend/internal/db/mapper/precompiled_object_mapper.go

```diff
@@ -114,7 +124,55 @@ func (pom *PrecompiledObjectMapper) ToPrecompiledObj(exampleDTO *dto.ExampleDTO)
 		Sdk:             exampleDTO.GetSDK(),
 		Complexity:      exampleDTO.GetComplexity(),
 		Tags:            exampleDTO.Example.Tags,
+		Emulator:        exampleDTO.GetEmulator(),
+		Datasets:        exampleDTO.GetDatasets(),
+	}
+}
+
```

Comment:
having at least minimal function/method descriptions is nice

Actually, I'd consider not using naked `map[string]smth`-like objects, they are counter-intuitive: it's hard to guess what is the meaning of a key

Better to type them explicitly:
`type DatasetById map[string]Dataset`